version.sbt

```diff
@@ -1 +1 @@
-version in ThisBuild := "0.0.47-SNAPSHOT"
+version in ThisBuild := "0.0.48-SNAPSHOT"
```

Comment:
Should this be the snapshot version?

Reply:
Yeah, it's snapshot when you're developing, when it releases, it releases the full version.